src/utils/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios"
+
+const api = axios.create({
+    baseURL: "https://nc-news-p6kr.onrender.com/api/"
+})
+
+export default api
```

Comment:
Code like:
``` js
api.get(`articles/${article_id}/comments`)
await api.patch(`articles/${article_id}`, {inc_votes: amount})
await api.get(`articles/${article_id}`)
```

all need to be in the api.js file. This is where you make functions that do the API call. That way you would only need to invoke a function in the component files and that will return the data

e.g. 
```
// src/utils/api.js

getAllArticles = () => {
 // do api.get(`articles`) and grab the data and return that from function
}

// in a component file

getAllArticles().then(articles => ...)
```